src/main.tsx

```diff
@@ -96,21 +95,19 @@ function renderCommentsUi(
 ): React.ReactElement {
     let {
         commentsEnabled,
-        showResolvedComments,
         user
     } = store.getState().settings;
     let commentsToRender = comments;
 
     if (!commentsEnabled || !user) {
         commentsToRender = [];
-    } else if (!showResolvedComments) {
-        // Hide all resolved comments unless they were resolved this session
-        commentsToRender = commentsToRender.filter(comment => {
-            return !(
-                comment.resolvedAt !== null && !comment.resolvedThisSession
-            );
-        });
     }
+    // Hide all resolved/deleted comments
+    commentsToRender = commentsToRender.filter(comment => {
```

Comment:
1. Use one liner for this. It's perfectly readable without trillion of curly braces and return statements.
2. If `deleted` can be a boolean only, then use `!deleted`.
3. If `deleted` can also be undefined (`?`) or other value, then use `deleted === false` as you do. Remember, strict equality check.
```suggestion
    commentsToRender = commentsToRender.filter(comment => !comment.deleted);
```

No harm using destruction syntax too if you fancy, no preference here. I tend to abuse it. 

```suggestion
    commentsToRender = commentsToRender.filter(({ deleted }) => !deleted);
```

Reply:
Ooh, nice, didn't think of using destructuring syntax that way. Changed in latest commits.